config/controller.yaml

```diff
@@ -35,6 +35,7 @@ spec:
           "-git-image", "github.com/tektoncd/pipeline/cmd/git-init",
           "-nop-image", "github.com/tektoncd/pipeline/cmd/nop",
           "-bash-noop-image", "github.com/tektoncd/pipeline/cmd/bash",
+          "-bash-working-dir-image", "github.com/tektoncd/pipeline/cmd/bash",
```

Comment:
Why not using `bash-noop-image` instead of having a new flag ? (we were also thinking of renaming this one as the `noop` make it more scarier than it is)

Reply:
We could use the noop image, though my thinking was that it's nice to have separate arguments for users to override if necessary. I.e. if for some reason in a user's setup they need to override the image that's used for creating working directories, e.g. to have their own custom logic run, but don't want to change it for the current use of bash-noop-image (inside of the artifact setup). Thoughts?

Reply:
That SGTM @dicarlo2

Reply:
(besides @vdemeester I think we can actually get rid of the noop-image now that we're not using init containers right? i'm having deja vu like we already discussed this... :thinking::thinking::thinking:)

Reply:
(ahahahaa that was a different image and you already removed it @dicarlo2 :rofl: :rofl: :rofl: https://github.com/tektoncd/pipeline/pull/715 thank you :pray: ... meanwhile we have got to get `noop` out of the name of that flag so ppl like me dont get confuse...)

Reply:
... and #715 hasn't been merged yet ... i'm going to catch up eventually i promise...